e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -1124,6 +1128,12 @@ func (s *IbcEurekaTestSuite) ICS20TimeoutPacketFromEthereumTest(
 			s.Require().NoError(err)
 			s.Require().Equal(ethtypes.ReceiptStatusSuccessful, receipt.Status)
 
+			if i == 0 {
```

Comment:
When we do slightly funky stuff like this (which I get after reading the rest of the code 40 lines down), I prefer to have a small comment to justify it.